packages/backend-api/integration/disintegrate.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+INTEGRATION_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+PROJECT_BASE_PATH=`[ -d "$PWD/project-base" ] && echo "$PWD/project-base" || echo "$PWD"`
```

Comment:
what about to make it consistent with `$(....)` instead of `` `.....` `` or vice versa

Reply:
To be honest, I don't understand the difference between these notations in bash. Please just tell me "do this" and I'll do it :-)